ncm-metaconfig/src/main/perl/metaconfig.pm

```diff
@@ -153,19 +153,20 @@ use LC::Exception;
 use EDG::WP4::CCM::TextRender 18.6.1;
 use CAF::Service;
 use EDG::WP4::CCM::Path qw(unescape);
+use Text::ParseWords qw(old_shellwords);
```

Comment:
Isn't is a bit risky to take a dependency on old_shellwords which is [deprecated](https://metacpan.org/pod/Text::ParseWords)? Why to use `shellwords`?

Reply:
how did i miss the `shellwords`? i already foudn it very odd.  i'll defintely change this.

Reply:
Did `shellwords` not work after all?

Reply:
oopsie, more comments i missed. i'll make the changes, can you do a `perl -e 'use Text::ParseWords qw(shellwords)'` to make sure it exists on el6?